app/experimenter/templates/experiments/section_branches.html

```diff
@@ -24,21 +24,36 @@
     {% endif %}
   {% endif %}
   {% for variant in experiment.variants.all %}
-    <div class="border-left-grey pl-3 my-3">
+    <div class=" pl-3 my-3">
       <h5>{{ variant.ratio }}% {{ variant.name }}</h5>
       {% if experiment.is_multi_pref %}
         <p><strong>Prefs:</strong></p>
+        <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Value</th>
+            <th scope="col">Type</th>
+            <th scope="col">Branch</th>
+          </tr>
+        </thead>
+        <tbody>
         {% for preference in variant.preferences.all %}
-          <div class="border-left-grey pl-3 ml-2 mb-3">
-           {% if preference.is_json_string_type %}
-            <p><strong> {{ preference.pref_name }} = <pre class="text-info">{{ preference.pref_value|as_json }}</pre></strong></p>
-           {% else %}
-            <p><strong> {{ preference.pref_name }} = <span class="text-info">{{ preference.pref_value }}</span></strong></p>
-           {% endif %}
-           <p>Pref Type: <span class="text-info">{{ preference.pref_type }}</span></p>
-           <p>Pref Branch: <span class="text-info">{{ preference.pref_branch }}</span></p>
-          </div>
+          <tr>
+            <td>{{ preference.pref_name }}</td>
+            <td>
+            {% if preference.is_json_string_type %}
+              <pre class="text-info">{{ preference.pref_value|as_json }}</pre>     
+            {% else %}
+              <pre class="text-info">{{ preference.pref_value }}</pre>
+            {% endif %}
```

Comment:
Don't need to repeat the markup at all

```html
<pre class="text-info">
  {% if preference.is_json_string_type %}
    {{ preference.pref_value|as_json }}  
  {% else %}
    {{ preference.pref_value }}
  {% endif %}
</pre>
```

Reply:
A little confused if you mean to get rid of the `<td>` tag too? or no? there's so weird white spacing...?
![Screen Shot 2020-05-05 at 7 37 06 PM](https://user-images.githubusercontent.com/25379936/81133915-f0758c80-8f07-11ea-8162-efb724c07721.png)

Reply:
No like these things should be equivalent:

```html
            <td>
            {% if preference.is_json_string_type %}
              <pre class="text-info">{{ preference.pref_value|as_json }}</pre>     
            {% else %}
              <pre class="text-info">{{ preference.pref_value }}</pre>
            {% endif %}
```

```html
<td>
  <pre class="text-info">
    {% if preference.is_json_string_type %}
      {{ preference.pref_value|as_json }}  
    {% else %}
      {{ preference.pref_value }}
    {% endif %}
  </pre>
</td>
```

Reply:
Just without repeating the markup

Reply:
idk if it's a me problem, but I'm still getting the weird white spacing

Reply:
Oh it's becuase unlike other HTML tags, the `<pre>` tag preserves whitespace!  So for this to work you'd have to stick the whole if statement on a single line.... Okay nevermind me, forget everything I said, put it back to the way you head it, that worked, I should've just let it alone.  Sorry @tiftran ! This is great, thank you!